integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -375,12 +359,15 @@ def export(self, **kwargs):  # pylint: disable=R0915
                         grade=grade_from_api,
                         is_passing=is_passing_from_api,
                         course_id=course_id,
-                        user_id=enterprise_enrollment.enterprise_customer_user.user_id,
+                        user_id=enterprise_user_id,
                         enterprise=enterprise_enrollment.enterprise_customer_user.enterprise_customer.slug
                     ),
                     channel_name=channel_name,
                     enterprise_customer_identifier=self.enterprise_customer.name
                 )
+
+            # todo: we need to refactor so single learner is not so mixed in with bulk case
```

Comment:
Please do not leave TODOs in code: file a ticket and leave the ticket here if necessary.